homeassistant/components/alexa/handlers.py

```diff
@@ -870,8 +876,9 @@ async def async_api_adjust_target_temp(
     """Process an adjust target temperature request."""
     data: dict[str, Any]
     entity = directive.entity
-    min_temp = entity.attributes[climate.ATTR_MIN_TEMP]
-    max_temp = entity.attributes[climate.ATTR_MAX_TEMP]
+    # Shared code for both climate and water heater devices
```

Comment:
Why is this comment here, shouldn't it be in the docstring for `async_api_adjust_target_temp` or is it there to justify the magic strings?

Reply:
Moved to docstr